storybook/qmlTests/tests/tst_OwnerTokenWelcomeView.qml

```diff
@@ -0,0 +1,82 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import QtTest 1.15
+
+import StatusQ 0.1
+
+import SortFilterProxyModel 0.2
+import AppLayouts.Communities.views 1.0
+
+import Models 1.0
+
+Item {
+
+    id: root
+    width: 600
+    height: 800
```

Comment:
This fixed the test case. Clicking next isn't working because it's out of the screen

```suggestion
    height: controlUnderTest ? controlUnderTest.implicitHeight : 800
```

Reply:
indeed, thanks for hint !